packages/create-plugin/src/commands/generate.plopfile.ts

```diff
@@ -113,6 +113,22 @@ export default function (plop: NodePlopAPI) {
         ? getActionsForTemplateFolder({ folderPath: backendFolderPath, exportPath })
         : [];
 
+      // Common, pluginType and backend actions can contain different templates for the same destination.
+      // This filtering removes the duplicate file additions to prevent plop erroring and makes sure the
+      // correct template is scaffolded.
+      // Note that the order is reversed so backend > pluginType > common
+      const pluginActions = [...backendActions, ...pluginTypeSpecificActions, ...commonActions].reduce((acc, file) => {
+        const actionExists = acc.find((f) => f.path === file.path);
+        // return early to prevent multiple add type actions being added to the array
+        if (actionExists && actionExists.type === 'add' && file.type === 'add') {
+          return acc;
+        }
+        acc.push(file);
+        return acc;
+      }, []);
+
+      console.log([...backendActions, ...pluginTypeSpecificActions, ...commonActions].length, pluginActions.length);
```

Comment:
extra console log?

Reply:
Oops. Tis gone now. Thanks for pointing this out!